### ISSUE TITLE
High CPU usage ( and custom inputs dots not drawn ) 

### ISSUE THREAD

**Issue by JorgeR81** [chrisgoringe/cg-use-everywhere#238]
I have higher CPU usage, with use-everywhere nodes.
This is more noticeable if I also use the Controller.

And sometimes the custom inputs dots are not drawn ( even without the Controller ).
With simpler workflows they are drawn, but disappear if I move or collapse/expand the node. 

![ae](https://github.com/user-attachments/assets/45193962-7e69-4656-801f-96d5d0cbe233)

**Comment by chrisgoringe** [comment]
There has been a change in the way Comfy draws nodes (that's why the controller was changing the headers). I'll need to investigate.

**Comment by JorgeR81** [comment]
The node highlight seems to break the "wireless dot".

This bug has an inconsistent behavior. Sometimes the "wireless dots" are drawn, and sometimes not.

But I tried a new (default) workflow, with a single AE node, and I was able to find a pattern.

---

- When I click the node with the "wireless dot", it stops being drawn.

![ae2](https://github.com/user-attachments/assets/1adc9d47-6049-457e-83e4-126287ebbfff)

---

- And when I click the AE node, the "wireless dot" is drawn again.
This works every time. It's like a switch !

![ae1](https://github.com/user-attachments/assets/92e9ea73-db17-46df-aa99-f296e978ccd3)
 
---

- Clicking on the other nodes in the workflow, does not change whether the "wireless dot" is drawn or not. 
( it will not change the state of the "swicth" )

---

If I add a 2nd AE node, the system becomes more complex.
Because, sometimes the 1st AE node acts as the "switch" that fixes the 2nd wireless dot.

But one thing remains the same:

- **If I click a node with a "wireless dot", the "wireless dot" stops being drawn.** 

![d2](https://github.com/user-attachments/assets/8eb9d896-5409-4ecd-a82f-8b58098d730c)
